pkg/collector/container_cgroup_collector.go

```diff
@@ -25,15 +25,34 @@ import (
 
 // updateCgroupMetrics adds container-level cgroup data
 func (c *Collector) updateCgroupMetrics() {
-	klog.V(5).Infof("overall cgroup stats %v", cgroup.SliceHandlerInstance)
-
-	for containerID := range c.ContainersMetrics {
-		cgroup.TryInitStatReaders(containerID)
-		cgroupFSStandardStats := cgroup.GetStandardStat(containerID)
-		for cgroupFSKey, cgroupFSValue := range cgroupFSStandardStats {
-			readVal := cgroupFSValue.(uint64)
-			if _, ok := c.ContainersMetrics[containerID].CgroupFSStats[cgroupFSKey]; ok {
-				c.ContainersMetrics[containerID].CgroupFSStats[cgroupFSKey].SetAggrStat(containerID, readVal)
+	for key := range c.ContainersMetrics {
+		if c.ContainersMetrics[key].PID == 0 {
+			klog.V(3).Infof("PID 0 does not have cgroup")
+			return
```

Comment:
return or continue?

Reply:
@marceloamaral , this one as blocking with PR merge from my point of view.

Reply:
@SamYuan1990 actually, the message is misleading I will update it.
We cannot get cgroup metrics for the pid 0. Basically because there is no `/proc/0/cgroup`. 

So, i will update the message to `klog.V(3).Infof("PID 0 does not have cgroup metrics")`

Reply:
Good catch, it should be a `continue` not a `return`.

Reply:
my focus is on the `return` here, if the PID is 0 and the function is returned. Hence other PID will be skipped?
in other word, do we have a case which `c.ContainersMetrics[0].PID==0` and `c.ContainersMetrics[1].PID==1` in same c.ContainersMetrics array?

Reply:
I have actually updated it to `continue`